<launch>

    <arg name="headless" default="false"/>
    <arg name="gui" default="true"/>
    <arg name="enable_logging" default="false"/>
    <arg name="enable_ground_truth" default="false"/>
    <arg name="log_file" default="ardrone"/>

    <include file="$(find rotors_gazebo)/launch/ardrone_empty_world_with_joy.launch">
        <arg name="headless" value="$(arg headless)"/>
        <arg name="gui" value="$(arg gui)"/>
        <arg name="enable_logging" value="$(arg enable_logging)" />
        <arg name="enable_ground_truth" value="$(arg enable_ground_truth)" />
        <arg name="log_file" value="$(arg log_file)"/>
    </include>
    <include file="$(find px4)/launch/ardrone.launch" />

</launch>
